.github/workflows/ci.yml

```diff
@@ -24,6 +24,10 @@ jobs:
         run: yarn install --pure-lockfile
       - name: TypeScript check
         run: yarn ts:check
+      - name: Make envfile
+        uses: SpicyPizza/create-envfile@v1.3
+        with:
+          envkey_JSON_BIN_X_MASTER_KEY: ${{ secrets.JSON_BIN_X_MASTER_KEY }}
```

Comment:
is it still required?

Reply:
No, I removed it also from secrets on the github. Also renamed it variable in code to `JSON_BIN_X_ACCESS_KEY`.